blog/views.py

```
@@ -1,10 +1,11 @@
 import logging
 import pprint
+import typing
 
 from django.contrib.auth.decorators import login_not_required
 from django.http import HttpResponse, HttpResponseRedirect
```

Ruff (0.8.2)

`django.http.HttpResponseRedirect` imported but unused

Remove unused import: `django.http.HttpResponseRedirect`

(F401)